oka/chap4/knock37.py

```diff
@@ -0,0 +1,28 @@
+import knock30
+from collections import Counter
+from matplotlib import pyplot, rcParams
```

Comment:
```suggestion
import matplotlib.pyplot as plt
from matplotlib import rcParams
```

慣習的に`matplotlib.pyplot`は`plt`という名前で宣言することが多いです．
ご参考までに．

他には，下のようなものとかはデータ分析でよく使われているかも．
```
import numpy as np
import datetime as dt
import torch.nn as nn
```